<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textbrow_Test">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>116</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_Print">
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_setText">
      <property name="text">
       <string>SetText</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_appendText">
      <property name="text">
       <string>AppendText</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
